packages/office-ui-fabric-react/src/utilities/selection/SelectionZone.tsx

```diff
@@ -139,6 +139,8 @@ export class SelectionZone extends BaseComponent<ISelectionZoneProps, {}> {
         } else {
           this._onItemSurfaceClick(ev, index);
         }
+      } else {
+        selection.setAllSelected(false);
```

Comment:
This line I think is unnecessary; we don't need to clear selection when focus is set on a header.